proposal.md

```diff
@@ -0,0 +1,51 @@
+## Summary
+
+A latent space is a low-dimensional representation of meaningful sources of variability in a dataset.
+Latent spaces can be sampled to provide meaningful interpolations between examples.
+Latent spaces of image datasets can be used to transition a picture of a young man to a female representation or to an older version of the subject.
+Biologically relevant latent spaces provide an opportunity for hypothesis generation on a genome-wide scale.
+For example, we could imagine that we might predict how gene expression would change over time, with drug treatment, in the context of a specific disease, or a combination of these factors.
+Certain deep learning methods are capable of constructing meaningful latent spaces in other fields.
+Our objective is to determine whether or not these methods can be trained with single cell expression data and the extent to which such methods define biologically-relevant latent spaces.
+
+## Aims
+
+**Aim 1: Develop proof-of-concept deep learning methods to learn a latent space from single cell transcriptomic data from the HCA.**
+
+**Aim 2: Apply dimensionality reduction techniques that are not based on deep learning to learn low-dimensional representations of single cell transcriptomic data from the HCA.**
+
+**Aim 3: Evaluate the extent to which low-dimensional representations of single cell data learned by various methods can be used to decompose bulk tissues in the context of rheumatic diseases.**
+
+## Prior Contributions / Preliminary Results (not required)
+
+* ADAGE/eADAGE
+* GANs + clinical data
+* Variational Autoencoders
+* Scleroderma pubs -> cell lineage importance
```

Comment:
@jaclyn-taroni : anything that we can very quickly summarize? _Total_ proposal length (all sections) = 1600 words. Might even be just to say: cell lineages matter for this disease. Reasonable place to examine existing bulk data in disease context?